src/Jobs/SyncMailAccountJob.php

```diff
@@ -60,15 +62,13 @@ public function handle(): void
             ->where('mail_account_id', $this->mailAccount->id)
             ->whereIntegerNotInRaw('id', array_values($this->folderIds))
             ->get('id')
-            ->each(
-                fn (MailFolder $folder) => DeleteMailFolder::make(['id' => $folder->id])->validate()->execute()
-            );
+            ->each(fn (MailFolder $folder) => DeleteMailFolder::make(['id' => $folder->id])->validate()->execute());
 
         if ($this->onlyFolders) {
             return;
         }
 
-        foreach ($folders as $folder) {
+        foreach ($folders->reverse() as $folder) {
```

Comment:
why reverse?

Reply:
the imap package fetches in reversed order compared to any regular client, so the inbox is at bottom not at top